### DISCUSSION TITLE
Using chart

### DISCUSSION THREAD

**Discussion by Noorani-MM** [nuxt/nuxt#21754] [Questions] [answered]
## Version 

Nuxt : 3.3.1

<hr />

Hi there 👋
Recently I start frontend development.  And now I want to have stacked bar chart in my project. What packages and resources do you recommend?

**Comment by MuhammadM1998** [comment, marked as answer]
I've used Chart.js and its cool. It has multiple official tutorials on excactly what you need. Check this [Starter Repo](https://stackblitz.com/edit/nuxt-starter-sgsm9b?file=package.json) where you can see how to install and configure it with Nuxt 3.

- [Docs: Stacked Bar Chart](https://www.chartjs.org/docs/latest/samples/bar/stacked.html)
- [Video: How to Create 100% Stacked Bar Chart in Chart JS](https://www.youtube.com/watch?v=ln-NKR2mBQY) 
- [Video: Stacked Bar Chart in ChartJS](https://www.youtube.com/watch?v=LF9NiVS6-Hc)

Apex chart has a good reputation too if you want to check it out.

**Reply by Noorani-MM** [reply]
Thank you 💯

**Reply by MuhammadM1998** [reply]
You're welcome! If you found the answer helpful please consider choosing it to mark the question answered
